src/main/kotlin/ai/blindspot/ktoolz/extensions/OtherExtensions.kt

```diff
@@ -87,3 +87,45 @@ inline fun <T : Any> T.applyIf(shouldApply: Boolean, block: T.() -> Unit): T {
     return this
 }
 
+/**
+ * Creates a string like "className(description)", for example "Double(42.0)"
+ * Useful either for implementing .toString() override.
```

Comment:
Useful ~~either~~ e.g. for. 

Either should have or part, it is better to use e.g. here